classifier/main.py

```diff
@@ -34,15 +35,25 @@
 logger = logging_client.logger(log_name)
 
 # GCP resources used in this module
+bq_client = bigquery.Client(project=PROJECT_ID)
 bq_dataset = f"{SURVEY}_alerts"
 ps_topic = f"{SURVEY}-SuperNNova"
 if TESTID != "False":  # attach the testid to the names
     bq_dataset = f"{bq_dataset}_{TESTID}"
     ps_topic = f"{ps_topic}-{TESTID}"
 bq_table = f"{bq_dataset}.SuperNNova"
+bq_table_id=f"{PROJECT_ID}.{bq_table}"
```

Comment:
`bq_table` is only used once so let's simplify the code and combine these into one line. 

```suggestion
bq_table_id=f"{PROJECT_ID}.{bq_dataset}.SuperNNova"
```